src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -1326,6 +1331,11 @@ bool LottieParser::parseEffect(LottieEffect* effect, void(LottieParser::*func)(L
             else if (property && KEY_AS("mn")) property->mn = djb2Encode(getString());
             else skip();
         }
+
+        if (sid && property) {
```

Comment:
Please consider this: 
`if (sid && property) registerSlot(effect, sid, property);`